skyportal/handlers/api/photometry.py

```diff
@@ -1418,12 +1456,51 @@ def put(self):
                                 f'Adding streams {stream_ids_update} to photometry {duplicate.id}'
                             )
 
+                    # if ignoring flux deduplication, do we update the value(s) of the existing photometry?
+                    # ALSO, we only allow updating rows when the existing row has an origin specified, not when it's None
+                    if (
+                        ignore_flux
+                        and ignore_flux_replace
+                        and str(duplicate.origin).strip().lower()
+                        not in ['none', '', 'nan', 'null']
+                    ):
+                        # we might have more than one datapoint at a given: jd, instrument_id, filter, and origin
+                        # that do have different flux values. We can't update the flux/fluxerr for all of them,
+                        # because this will conflict with the deduplication index we have and raise an error
+                        # so we only want to update the first one we find
+                        # for that, we compute a hash of the jd, instrument_id, filter, and origin everytime we update a row
+                        # store it, and make sure we don't update a row with the same hash twice
```

Comment:
Also unclear; can we better document when this situation arises?

Reply:
I updated that comment, though the docstring should now explain what's happening (might not even need another comment there)